src/components/Amalgamation/ResultingBusinessName.vue

```diff
@@ -94,6 +94,8 @@ export default class ResultingBusinessName extends Mixins(AmalgamationMixin, Nam
   @Getter(useStore) getNameRequestNumber!: string
   @Getter(useStore) getShowErrors!: boolean
 
+  // @Action(useStore) setCorrectNameOption!: (x: CorrectNameOptions) => void
```

Comment:
This now comes via a mixin, so I commented it out to pre-empt a VS Code warning.